tests/test-requirement.txt

```diff
@@ -7,4 +7,5 @@ requests_mock
 types-requests
 pytest_mock
 ruff
-semantic_version
\ No newline at end of file
+semantic_version
```

Comment:
Why is semantic_version in test requirements?

Reply:
Ok.